python/hail/methods/qc.py

```diff
@@ -286,7 +273,8 @@ def concordance(left, right):
         per column key, and a table with concordance statistics per row key.
     """
 
-    right = _verify_biallelic(right, "concordance, right")
+    left = require_biallelic(right, "concordance, left")
```

Comment:
typo here! should be `left`. Maybe I should write a better test that doesn't have left == right ...